_config.yml

```diff
@@ -13,3 +13,4 @@ exclude: [
   "node_modules"
 ]
 livereload: true
+exclude: [vendor]
```

Comment:
Had to add this to the config to get it running locally for me.

Reply:
No problem in adding this, we might want to add [even more files & folders](https://github.com/jekyll/jekyll/blob/6855200ebda6c0e33f487da69e4e02ec3d8286b7/lib/site_template/_config.yml#L37-L55).